src/components/NavBar/style.scss

```
@@ -0,0 +1,27 @@
+@use '@/style/package' as *;
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  font-size: 15px;
```

크롬 익스텐션이라 반응형에 크게 민감할 필요는 없지만 반응형을 위해 rem을 쓰고 있어
html에 10px 걸어놨고, 우리는 10px === 1rem 으로 쓰면 돼

```suggestion
  font-size: 1.5rem;
```

---

완!